backend/controllers/image.js

```diff
@@ -74,4 +74,58 @@ exports.getAllStuff = (req, res) => { // call the get method, which adds a route
   Image.find() // call the find method, which returns a promise, which resolves to an array of all the Image objects in the database
     .then(images => res.status(200).json(images)) // call the then method, which adds a callback function to the promise, to handle the success case
     .catch(error => res.status(400).json({ error: error })); // call the catch method, which adds a callback function to the promise, to handle the failure case
-}
\ No newline at end of file
+}
+
+
+exports.Likes = async (req, res) => {
```

Comment:
I would change the name of this function as likes is very ambiguous and does not really describe what the function does.